### PR TITLE
README.md / resource-pack update

### PR DESCRIPTION
**Pull request by tp-duolingo**
Fixed some markdown issues and rewrote a tiny amount of it

### CODE REVIEW

File: content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md [hunk 1]
```diff
@@ -10,38 +10,42 @@ author: "Kuroi Jigoku & Mocab"
 
 ## Introduction
 
-Adding a resource pack to your Minecraft server can greatly enrich the player experience and maintain server aesthetics by altering the game's textures, sounds and font without altering the game's code directly. Resource packs are client side modifications and are therefore sent to individual players from the server to be stored and run on their own device.
+- Adding a resource pack to your Minecraft server can greatly enrich the player experience and maintain server aesthetics by altering the game's textures, sounds and font without altering the game's code directly. Resource packs are client side modifications and are therefore sent to individual players from the server to be stored and run on their own device.
 
-This guide will demonstrate how to add a server-wide resource pack popup to allow users to enable and use a specified resource pack, or alternatively, enforce it for all players.
+- This guide will demonstrate how to add a server-wide resource pack popup to allow users to enable and use a specified resource pack, or alternatively, enforce it for all players.
```
Comment (iamdeka) [inline content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md:15]: ⁉️

File: content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md [hunk 2]
```diff
@@ -10,38 +10,42 @@ author: "Kuroi Jigoku & Mocab"
...
--   [Curseforge](https://www.curseforge.com/minecraft-bedrock "Known for hosting mods and addons for various games, CurseForge also has a section for Minecraft Bedrock Edition resource packs.")
+- [Planet Minecraft](https://www.planetminecraft.com/ "A popular platform for sharing Minecraft content, including resource packs, maps, and skins. The site has a dedicated section for Bedrock Edition.")
+- [MCPE DL](https://mcpedl.com/ "A website specifically focused on Minecraft Pocket Edition and Bedrock Edition, offering a wide range of resource packs, mods, maps, and more.")
+- [Minecraft Forum](https://www.minecraftforum.net/ "A long-standing community forum for Minecraft players, featuring a section dedicated to resource packs for all editions, including Bedrock.")
+- [Curseforge](https://www.curseforge.com/minecraft-bedrock "Known for hosting mods and addons for various games, CurseForge also has a section for Minecraft Bedrock Edition resource packs.")
 
-You can also upload your own custom resource pack as well.
+You can also upload your own custom resource pack too.
```
Comment (iamdeka) [inline content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md:26]: ⁉️

File: content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md [hunk 3]
```diff
@@ -10,38 +10,42 @@ author: "Kuroi Jigoku & Mocab"
...
 
 > Ensure the resource pack is compatible with the version of Minecraft used and any addons or behavior packs installed. Incompatible packs may cause crashes or visual glitches.
 
 ## Installation
 
-### Obtaining the Pack Metadata
+### Obtaining The Pack Metadata
 
-1. Download the resource pack. If the file is in the `.mcpack` format, rename the file extension to the `.zip` format.
+- Download the resource pack.
```
Comment (iamdeka) [inline content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md:36]: ⁉️

**Review by iamdeka — CHANGES_REQUESTED** [review]

**Review by iamdeka — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -2,72 +2,67 @@
...
-    -   [✒️ Markdown](https://github.com/FalixNodes-Software/Knowledge-base#%EF%B8%8F-markdown)
-        -   [Headers](https://github.com/FalixNodes-Software/Knowledge-base#headers)
-        -   [Images](https://github.com/FalixNodes-Software/Knowledge-base#images)
-        -   [Blockquote](https://github.com/FalixNodes-Software/Knowledge-base#blockquote)
-        -   [Video](https://github.com/FalixNodes-Software/Knowledge-base#video)
-    -   [📢️ Publishing](https://github.com/FalixNodes-Software/Knowledge-base#%EF%B8%8F-publishing)
-    -   [🏗️ Building and Testing Locally](https://github.com/FalixNodes-Software/Knowledge-base#%EF%B8%8F-building-and-testing-locally)
-        -   [Running Natively](https://github.com/FalixNodes-Software/Knowledge-base#method-1-running-natively)
-        -   [Running with Docker](https://github.com/FalixNodes-Software/Knowledge-base#method-2-running-with-docker)
+You're currently viewing the source code that makes up the [Knowledge Base](https://kb.falixnodes.net/) of FalixNodes. It's all built on Jekyll, a static website generator we use.
```
[the remark this answers is not in the document]
Comment (tp-duolingo) [reply README.md:5]: there were unnecessary lines

File: content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md (hunk 1 above)
Comment (tp-duolingo) [reply content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md]: added a bullet so it looks better

File: content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md (hunk 2 above)
Comment (tp-duolingo) [reply content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md]: why did you comment on this ❔

File: content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md (hunk 3 above)
Comment (tp-duolingo) [reply content/_posts/minecraft/bedrock/2024-07-13-resource-pack.md]: why did you comment on this part 2

**Comment by iamdeka** [comment]
Make a separate PR for readme edit and for each of the article edits.

**Comment by tp-duolingo** [comment]
Since when you could edit pr's 🤔 But alright, taken into consideration

**Comment by iamdeka** [comment]
> Since when you could edit pr's 🤔

Create a different branch in your fork for each pr

**Pull request closed without merging**